cmd/watch-records.go

```diff
@@ -1,8 +1,8 @@
-// Package cmd implements fyde-cli commands
+// Package cmd implements access-cli commands
 package cmd
 
 /*
-Copyright © 2019 Fyde, Inc. <hello@fyde.com>
+Copyright © 2020 Barracuda Networks, Inc. <hello@barracuda.com>
```

Comment:
Is hello@barracuda.com a valid email address?

Reply:
Fixed